modules/ppcp-settings/src/Endpoint/ConnectManualRestEndpoint.php

```diff
@@ -134,6 +135,18 @@ public function connect_manual( WP_REST_Request $request ) : WP_REST_Response {
 			);
 		}
 
+		$settings = new GeneralSettings();
```

Comment:
We want to reduce dependencies to make the classes more reusable, i.e., please use dependency injection for this (the plugin uses Modularity; services are defined in the `services.php` file).